flink-connectors/flink-connector-cassandra/src/main/resources/META-INF/NOTICE

```diff
@@ -6,9 +6,9 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt) 
 
-- com.datastax.cassandra:cassandra-driver-core:3.0.0
-- com.datastax.cassandra:cassandra-driver-mapping:3.0.0
-- com.google.guava:guava:18.0
+- com.datastax.cassandra:cassandra-driver-core:3.11.2
+- com.datastax.cassandra:cassandra-driver-mapping:3.11.2
+- com.google.guava:guava:19.0
 - io.netty:netty-buffer:4.1.70.Final
```

Comment:
eeeeeeeeeeeeeeeeeeeeeeeeeh

license check now complains that netty isn't being bundled, because the shade-plugin doesn't see it.
So I guess we'll just remove these entries.

Reply:
yet another check that I was not aware of :smile: and that I do not see in my local clean verify on cassandra module. Thanks for pointing out @zentol.